avsomr/avsomr.cpp

```diff
@@ -0,0 +1,368 @@
+#include "avsomr.h"
+
+#include "avslog.h"
+
+namespace {
+static int TRASH_GLYPH_SIZE{10};
+static int DEFAULT_SYSTEM_GAP{200};
+static float SYSTEM_GAP_PERCENT{0.55};
+}
+
+using namespace Ms::Avs;
+
+AvsOmr::AvsOmr()
+      {
+
+      }
+
+//---------------------------------------------------------
+//   resolve - calculates some elements and attributes
+//---------------------------------------------------------
+
+void AvsOmr::resolve()
+      {
+      auto firstMS = [](Sheet* sh) -> const MStack* {
```

Comment:
is this `->`a pointer? If so: no spaces before and after

Reply:
this is the syntax for specifying the return value from lambda